tasks/main.yml

```diff
@@ -24,6 +28,16 @@
   include_tasks: "{{ ansible_os_family }}{{ ansible_distribution_major_version }}.yml"
   when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "8"
 
+- name: rabbitmq.conf
+  template:
+    src: rabbitmq.conf.j2
+    dest: "{{ rabbitmq_cfg_file }}"
+    mode: 0644
+    owner: root
+    group: root
+  when: rabbitmq_cfg_settings
+  notify: restart rabbitmq
```

Comment:
Greetings!
Several edits are needed:
- Handler name should be start from capital letter
- Use Ansible.Builtin prefix on all tasks